From 2b72af15779549dc945b66725811588c2fff2859 Mon Sep 17 00:00:00 2001
From: Iain Lane <iain@orangesquash.org.uk>
Date: Thu, 4 May 2017 15:37:13 +0100
Subject: [PATCH 3/3] build: limba: Set limba as a dependency

We were failing to build the limba plugin because the dependency was
missing.
---
 plugins/limba/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/limba/meson.build b/plugins/limba/meson.build
index 55ccd445..e61a2db1 100644
--- a/plugins/limba/meson.build
+++ b/plugins/limba/meson.build
@@ -11,6 +11,6 @@ shared_module(
   install : true,
   install_dir: plugin_dir,
   c_args : cargs,
-  dependencies : plugin_libs
+  dependencies : [ plugin_libs, limba ]
   )
 endif
-- 
2.11.0

